Base/QTGUI/qSlicerApplication.cxx

```diff
@@ -179,6 +180,19 @@ void qSlicerApplicationPrivate::init()
 {
   Q_Q(qSlicerApplication);
 
+  qMRMLWidget::OpenGLProfileType openGLProfile = qMRMLWidget::OpenGLProfileDefault;
+  QString useOpenGLCompatibilityProfileApplicationSetting =
+    qSlicerApplication::application()->userSettings()->value("OpenGLProfile").toString().toLower();
```

Comment:
Is it valid to call `qSlicerApplication::application()->userSettings()` from within the constructor ?

Reply:
Yes, it works there, since the parent class constructor is already completed.

Reply:
Great. Trying now to build locally on linux